### PR TITLE
Modify gradle build for c2c with ballerina gradle plugin

### LINKED ISSUE #519: Migrate to Ballerina Gradle plugin
**Description:**
$Subject

**Describe your task(s)**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by Thevakumar-Luheerathan**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: ballerina/build.gradle [hunk 1]
```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Dilhasha) [inline ballerina/build.gradle:2]:
```suggestion
 * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: ballerina/build.gradle [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    dependencies {
+        classpath "io.ballerina:plugin-gradle:${project.ballerinaGradlePluginVersion}"
+    }
+}
+
+def packageName = "cloud"
+def packageOrg = "ballerina"
+def tomlVersion = stripBallerinaExtensionVersion("${project.version}")
+
+
```
Comment (Dilhasha) [inline ballerina/build.gradle:43]: Let's remove all unnecessary new lines

File: ballerina/build.gradle [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+
+description = 'Ballerina - Code2Cloud Extension Annotations'
+
+import org.apache.tools.ant.taskdefs.condition.Os
```
Comment (xlight05) [inline ballerina/build.gradle:22]: Repeated Import

File: ballerina/build.gradle [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+
+description = 'Ballerina - Code2Cloud Extension Annotations'
```
Comment (xlight05) [inline ballerina/build.gradle:20]: Lets remove the annotations part

File: ballerina/build.gradle [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+        classpath "io.ballerina:plugin-gradle:${project.ballerinaGradlePluginVersion}"
+    }
+}
+
+def packageName = "cloud"
+def packageOrg = "ballerina"
+def tomlVersion = stripBallerinaExtensionVersion("${project.version}")
+
+
+
```
Comment (xlight05) [inline ballerina/build.gradle:44]: Unwanted empty newlines

**Comment by Thevakumar-Luheerathan** [comment]
we will be merging this after release

**Pull request closed without merging**